### DISCUSSION TITLE
.

### DISCUSSION THREAD

**Discussion by dc932dc** [volantis-x/hexo-theme-volantis#826] [Q&A] [answered]

**Comment by BeaCox** [comment]
能否提供复现地址？

**Reply by Tikas** [reply]
可以把仓库 clone 一份到另一个仓库，让我们来复现复现

**Comment by BeaCox** [comment, marked as answer]
经过测试，在`Blog-source\node_modules\hexo-generator-json-content\`目录下存在多余的`node_modules`文件夹导致报错，删掉即可
